### DISCUSSION TITLE
Difficulty onboarding organization

### DISCUSSION THREAD

**Discussion by JohnStrunk** [renovatebot/renovate#23879] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I'm trying to enable the "forking-renovate" app for my organization: [backube](https://github.com/backube)
I have added the forking-renovate app. It appears in the list of installed apps, and I have enabled it for the [backube/snapscheduler](https://github.com/backube/snapscheduler) repo.

Problems:
- I did not get the onboarding PR
- I am unable to access the developer dashboard for backube.

When I log in to https://developer.mend.io/, I get:
![image](https://github.com/renovatebot/renovate/assets/6456608/4a5ab9c4-c693-40ac-bd53-64f11b16ffd9)

Clicking on the Backube tile produces:
![image](https://github.com/renovatebot/renovate/assets/6456608/e9f7d539-6973-4d5b-95a4-0bce448a8647)

**But, I am the owner of the Backube organization.**

### Workaround

- I am able to go directly to the page for snapscheduler (https://developer.mend.io/github/backube/snapscheduler) and interact with the dashboard there.
- From that page, I was able to manually enable renovate in the settings via local override and request a run. This created the onboarding PR in my repo.
- It looks like I need to manually trigger runs when I update the onboarding PR

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by nabeelsaabna** [comment]
Hi @JohnStrunk 
the first login/onboarding of the organization need to be completed by a user with admin permission on the organization level
do you have admin permissions on the organization ?

**Reply by JohnStrunk** [reply]
Yes. I am the owner of the org

**Reply by nabeelsaabna** [reply]
Is it happening also for the user account or only the `backube` org ?

**Comment by rarkins** [comment, marked as answer]
Hi @JohnStrunk the root cause was identified: the Forking Renovate app did not request access to read Org member permissions, which meant it couldn't validate you as owner/admin, which meant you couldn't onboard the org. We have now added this permission to the app, but existing users like yourself will need to do either of the following:
- Uninstall then reinstall the app, or
- Look for an email from GitHub requesting increased permissions, click the link, and then Approve

After that you should hopefully be promoted to onboard that org

**Reply by JohnStrunk** [reply]
That did it. Thank you!

**Comment by HonkingGoose** [comment]
> the root cause was identified: the Forking Renovate app did not request access to read Org member permissions, which meant it couldn't validate you as owner/admin, which meant you couldn't onboard the org. We have now added this permission to the app,

@rarkins The docs has a page explaining the permissions for the Mend Renovate App and Forking Renovate. [^1] The new "read Org member" permission is _not_ in the table. Do we need to update the docs? I don't know _where_ to put the new permission, or maybe we need to expand the _Why_ section for an existing permission?

[^1]: [Renovate docs, permissions for the Mend Renovate App and Forking Renovate](https://docs.renovatebot.com/security-and-permissions/#permissions)

**Reply by rarkins** [reply]
I think this might only be used by the Renovate App UI. Essentially, we allow admins to change Renovate settings for an org, so we need read access to be able to verify that they're an admin.
